src/server/req_rescq.c

```diff
@@ -583,7 +583,9 @@ req_confirmresv(struct batch_request *preq)
 	 */
 	if (is_being_altered)
 		free_resvNodes(presv);
-	rc = assign_resv_resc(presv, next_execvnode);
+	int svr_init;
+	svr_init = FALSE;
```

Comment:
Also declaring variables in this portion of the code will fail Windows Build. It looks like this is one of the reason appveyor is failing the windows build. Please look into the Windows build and also test it on Windows.